code/Test_definitions/Geofencing.feature

```diff
@@ -0,0 +1,118 @@
+#-
+# ---license-start
+# CAMARA Project
+# ---
+# Copyright (C) 2023 - 2024 Contributors | Deutsche Telekom AG to CAMARA a Series
+#
+# The contributor of this file confirms his sign-off for the
+# Developer Certificate of Origin
+#             (http://developercertificate.org).
+# ---
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ---license-end
+
+ @Geofencing 
+Feature: Operations to manage event subscription on geofencing events for leaving and entering in an area
+
+  @geofencing_01_Create_geofencing_subscription_for_a_device
+  Scenario:  Create geofencing subscription
+    Given they use the geofencing url
+    When they create subscription
+    Then Response code is 201
+
+  @geofencing_02_Operation_to_retrieve_list_of_subscriptions.--Not yet implemented
```

Comment:
I guess a space is needed before the double dash, at the moment the string "--Not" is perceived as part of the expression starting with '@'